docs/CONFIG.md

```diff
@@ -5,29 +5,63 @@ While most of EVE's configuration is done via receiving a configuration object f
 * `v2tlsbaseroot-certificates.pem` - contains the x509 root certificate to trust for the TLS to the controller when using the V2 API
 * `onboard.cert.pem` - onboarding certificate for the [initial registration](REGISTRATION.md) with the controller
 * `wpa_supplicant.conf` - a legacy way of configuring EVE's WiFi
 * `authorized_keys` - initial authorized SSH keys for accessing EVE's debug console
+* `bootstrap-config.pb`- initial device configuration used only until device is onboarded (see below for details)
 
-The initial content of these configuration files is stored in the EVE's source tree under [config](../config) folder. From there, these configuration files are baked into the EVE installer images. For the read-write bootable disk installer image these files can further be tweaked by mounting the CONFIG partition and editing those files directly on the installer image. This gives you an ability to take the default installer image and tweak it for your needs without re-building EVE from scratch (obviously this is not an option for a read-only ISO installer image). A typical workflow is to take an installer image from the official EVE build, flash it onto a USB flash drive, insert that USB flash drive into your desktop and edit file on the partition called EVE.
+The initial content of these configuration files is stored in the EVE's source tree under [config](../config) folder. From there, these configuration files are baked into the EVE installer images. For the read-write bootable disk installer image these files can further be tweaked by mounting the "EVE" partition and editing those files directly on the installer image. This gives you an ability to take the default installer image and tweak it for your needs without re-building EVE from scratch (obviously this is not an option for a read-only ISO installer image). A typical workflow is to take an installer image from the official EVE build, flash it onto a USB flash drive, insert that USB flash drive into your desktop and edit file on the partition called EVE.
 
-Once an installer has been run on an edge node, the content of the CONFIG partition found on the installer media is simply copied into the CONFIG partition on the edge node's local storage. From that point on, even though these files reside in the read-write CONFIG partition on the edge node, they can NOT be changed by the controller. The only option for tweaking these files is to have debug console access either via local terminal, remote ssh terminal or batch removable media configuration.
+Once an installer has been run on an edge node, the content of the EVE partition found on the installer media is simply copied into the CONFIG partition on the edge node's local storage. From that point on, these files reside in the *read-only* CONFIG partition on the edge node and can NOT be changed by the controller.
```

Comment:
Copy from CONFIG partition I assume.

Reply:
I'm not sure, but when I prepare USB installer I see that the partition with configs is labeled as EVE.

Reply:
You're correct; and the partition which has the rootfs has no label!

But to make this more clear, how about saying "the content of the partition with label EVE found on the installer media"?